Engine/ScriptAnalyzerEngine.csproj

```diff
@@ -44,6 +48,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\..\..\..\fbl_srv2_ci_mgmt.binaries.amd64chk\monad\System.Management.Automation.dll</HintPath>
```

Comment:
The project file in Rules also contain this hintpath. I think it may be auto generated by VS solution.